projects/plugins/jetpack/changelog/improve-subscriber-stats-on-editor-newsletter-panel

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: other
+
+- Show subscribers count from Jetpack for posts older than a day. This change is made to accurately show numbers on older posts.
+- Add a link which navigates to the Jetpack Email Stats page.
```

Comment:
The 2 lines won't necessarily work well when the general changelog is compiled from all the changelog entries. I would suggest merging this into a single line:

```suggestion
Subscriptions: adjust display of the number of subscribers in the Newsletter panels of the block editor for better usability.
```